src/main/resources/META-INF/aop.xml

```diff
@@ -19,17 +19,18 @@
 <aspectj>
           
    <aspects>
-       <!-- Configuration for Consistent Listing -->
+        <!-- Configuration for Consistent Listing --> 
         
-       <aspect name="com.netflix.bdp.s3mper.listing.ConsistentListingAspect"/>
-       <concrete-aspect name="com.netflix.bdp.s3mper.listing__ConsistentListing"
-                        extends="com.netflix.bdp.s3mper.listing.ConsistentListingAspect">
-           <pointcut name="init" expression="execution(* org.apache.hadoop.fs.FileSystem.initialize(..))"/>
-           <pointcut name="create" expression="execution(* org.apache.hadoop.fs.FileSystem.create(..)) ||
-                                                execution(* org.apache.hadoop.fs.FileSystem.mkdirs(..))"/>
-           <pointcut name="list" expression="execution(* org.apache.hadoop.fs.FileSystem.listStatus(..))"/>
-           <pointcut name="delete" expression="execution(* org.apache.hadoop.fs.FileSystem.delete(..))"/>
-       </concrete-aspect>
+        <aspect name="com.netflix.bdp.s3mper.listing.ConsistentListingAspect"/>
+        
+        <concrete-aspect name="com.netflix.bdp.s3mper.listing__ConsistentListing"
+                         extends="com.netflix.bdp.s3mper.listing.ConsistentListingAspect">
+            <pointcut name="init" expression="execution(* com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystemBase.initialize(..))"/>
+            <pointcut name="create" expression="execution(* com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystemBase.create(..)) ||
+                                                execution(* com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystemBase.mkdirs(..))"/>
+            <pointcut name="list" expression="execution(* com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystemBase.listStatus(..))"/>
+            <pointcut name="delete" expression="execution(* com.google.cloud.hadoop.fs.gcs.GoogleHadoopFileSystemBase.delete(..))"/>
+        </concrete-aspect>
```

Comment:
Can we have the pointcuts for both this *and* `NativeS3FileSystem`? That way the S3 stuff would continue to work.